api/tests/test_shipment.py

```diff
@@ -0,0 +1,136 @@
+            "order_key": "xxx",
+            "customer_note": "",
+            "shipping": {
+                "first_name": "John",
+                "last_name": "Lennon",
+                "address_1": "Beatles Street 66",
+                "address_2": "",
+                "postcode": "08032",
+                "city": "Barcelona",
+                "state": "Barcelona",
+                "country": "ES",
+            },
+            "billing": {"phone": "666666666", "email": "lennon@example.com"},
+            "line_items": [
+                {
+                    "id": 17,
+                    "quantity": 1,
+                    "name": "Suc Taronja 1l",
+                    "meta_data": [
+                        {
+                            "id": 172,
+                            "key": "_vendor_id",
+                            "value": "6",
+                            "display_key": "Store",
+                            "display_value": "Quèviure",
+                        }
+                    ],
+                }
+            ],
+        }
+
+        responses.add(
+            responses.GET,
+            f"{koiki.wcfmmp_host}/wp-json/wcfmmp/v1/settings/id/6",
+            status=200,
+            json={
+                "phone": "",
+                "address": {
+                    "street_1": "",
+                    "street_2": "",
+                    "city": "",
+                    "zip": "",
+                    "country": "ES",
+                    "state": "",
+                },
+            },
+        )
+
+        responses.add(
+            responses.GET,
+            "https://wp_testing_host/wp-json/wp/v2/users/6?context=edit",
+            status=200,
+            content_type="application/json",
+            json={
+                "id": 6,
+                "username": "Queviure",
+                "email": "test@test.es",
+                "roles": ["testrole"],
+            },
+        )
+
+    @responses.activate
+    @patch("koiki.email.EmailMessage", autospec=True)
+    def test_create_shipment_successful(self, mock_email):
+        responses.add(
+            responses.POST,
+            "https://testing_host/rekis/api/altaEnvios",
+            status=200,
+            json={
+                "respuesta": "101",
+                "mensaje": "OK",
+                "envios": [
+                    {
+                        "numPedido": "123",
+                        "codBarras": "yyy",
+                        "etiqueta": "abcd",
+                        "respuesta": "101",
+                        "mensaje": "OK",
+                    }
+                ],
+            },
+        )
+        serializer = OrderSerializer(data=self.data)
+        self.assertTrue(serializer.is_valid())
+        order = serializer.validated_data
+        mock_email.send.return_value = True
+        create_delivery(order)
+        self.assertEqual(Shipment.objects.all().count(), 1)
+        shipment = Shipment.objects.first()
```

Comment:
Some spacing between logical blocks would increase a bit the readability of this test. It's not easy to follow IMO.

Reply:
Fixed by: https://github.com/coopdevs/lazona_connector/pull/87/commits/a3e9a2a879597490c8e697e8ff3284d24840107a